src/psij/launchers/scripts/mpi_launch.sh

```diff
@@ -5,10 +5,19 @@ source $(dirname "$0")/launcher_lib.sh
 _PSI_J_PROCESS_COUNT="$1"
 shift
 
+IS_OPENMPI=0
+if mpirun -version | grep "Open MZI" >/dev/null 2>&1; then
```

Comment:
There is a typo (`MZI`)

Reply:
Uh, that's not a typo. I was testing both possibilities (string found and string not found) and forgot to revert. Will fix.